filestore/protoboards_test.go

```diff
@@ -0,0 +1,183 @@
+package filestore_test
```

Comment:
I might consider a different testing strategy using [test fixtures](https://dave.cheney.net/2016/05/10/test-fixtures-in-go).  Essentially, make a subdirectory called `testdata/protoboard` in the filestore directory and put the protoboard json there and use that data in the tests.

Here's an example of it in use: https://github.com/influxdata/kapacitor/blob/89828ffff6cf5cd4cb2b34bf883e134395f734de/server/server_helper_test.go#L88-L96

https://github.com/influxdata/kapacitor/tree/216b381639c513cf30c042f2f5ef54820b14e303/server/testdata

Reply:
why?